src/sentry/static/sentry/app/views/onboarding/integrationSetup.tsx

```diff
@@ -0,0 +1,390 @@
+            provider.name
+          )}
+        </p>
+        <p>
+          {t(
+            'This snippet includes an intentional error, so you can test that everything is working as soon as you set it up:'
+          )}
+        </p>
+        <div>
+          <CodeWrapper>
+            <code>
+              <TokenFunction>myUndefinedFunction</TokenFunction>
+              <TokenPunctuation>();</TokenPunctuation>
+            </code>
+          </CodeWrapper>
+        </div>
+        <p>
+          {t(
+            "If you're new to Sentry, use the email alert to access your account and complete a product tour."
+          )}
+        </p>
+        <p>
+          {t(
+            "If you're an existing user and have disabled alerts, you won't receive this email."
+          )}
+        </p>
+      </div>
+    );
+  }
+
+  renderPostInstallInstructions() {
+    const {organization, project, platform} = this.props;
+    const {provider, loadedPlatform} = this.state;
+    if (!project || !provider) {
+      return null;
+    }
+    const currentPlatform = loadedPlatform ?? platform ?? 'other';
+    return (
+      <React.Fragment>
+        <TitleContainer>
+          <StepHeading step={2}>
+            {t(
+              'Automatically instrument %s',
+              platforms.find(p => p.id === currentPlatform)?.name ?? ''
+            )}
+          </StepHeading>
+          <motion.div
+            variants={{
+              initial: {opacity: 0, x: 20},
+              animate: {opacity: 1, x: 0},
+              exit: {opacity: 0},
+            }}
+          >
+            <PlatformIcon size={48} format="lg" platform={currentPlatform} />
+          </motion.div>
+        </TitleContainer>
+        {this.renderPostInstallText()}
+        <FirstEventIndicator
+          organization={organization}
+          project={project}
+          eventType="error"
+        >
+          {({indicator, firstEventButton}) => (
+            <CTAFooter>
+              <Actions gap={2}>
+                {firstEventButton}
+                <Button
+                  external
+                  href={this.platformDocs}
+                  onClick={this.handleFullDocsClick}
+                >
+                  {t('View full documentation')}
+                </Button>
+              </Actions>
+              {indicator}
+            </CTAFooter>
+          )}
+        </FirstEventIndicator>
+        <CTASecondary>
+          {tct(
+            'Just want to poke around before getting too cozy with the SDK? [sample:View a sample event for this SDK] and finish setup later.',
+            {
+              sample: (
+                <CreateSampleEventButton
+                  project={project ?? undefined}
+                  source="onboarding"
+                  priority="link"
+                />
+              ),
+            }
+          )}
+        </CTASecondary>
+      </React.Fragment>
```

Comment:
Can we also pull this whole footer out into a component, looks like it's pretty much the same as what's in the doc setup right?